test/index.test.js

```diff
@@ -1958,6 +1958,133 @@ test('inline registration of component args - arrow function property lookup', a
   is(convert(src), esx)
 })
 
+
+test('React.Fragment', async ({ is }) => {
+  const src = [
+    `const React = require("react")`,
+    `const Cmp = () => <React.Fragment><div>Foo</div></React.Fragment>`,
+    `module.exports = Cmp`
+  ].join('\n')
+
+  const esx = [
+    `const esx = require('esx')()`,
```

Comment:
Not a big deal but avoid mixing “ and ‘ (stick with ‘)

Reply:
Ops... sorry about that :)